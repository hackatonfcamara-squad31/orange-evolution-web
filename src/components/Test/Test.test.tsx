import { render, screen, within } from '@testing-library/react'
import { describe, expect, it } from 'vitest'
import { Test } from '.'

describe('<Test />', () => {
  it('should render the Test heading', () => {
    const { container } = render(<Test />)

    const heading = within(screen.getByRole('heading', { name: /Test/i }))

    expect(heading).toBeDefined()

    expect(container.firstChild).toMatchSnapshot()
  })
})
