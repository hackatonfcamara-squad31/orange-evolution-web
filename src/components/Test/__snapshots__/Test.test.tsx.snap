// Vitest Snapshot v1

exports[`<Test /> > should render the Test heading 1`] = `
<div
  class="c-PJLV"
>
  <h1>
    Test
  </h1>
</div>
`;
